src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the beginning of the class definition.